pages/irac-2022.md

```diff
@@ -49,13 +47,79 @@ We have invited renowned speakers who will give an introductory overview for eac
+17:15-20:00 | Drinks & poster session
+
+### Saturday - June 26
+
+time        | presentation
+----------- | ------------
+**Session 2**{: .text-primary } | **Methodological advancements - classification and identification algorithms, radar technology, data infrastructure**{: .text-primary }
+ 9:00-9:30  | **Buler Jeff** - Applied radar aeroecology
+ 9:30-9:45	| **Bart Hoekstra** - Fireworks disturbance across bird communities
+09:45-10:00 | **William Kunin** - Tracking desert locust swarms using dual-polarisation weather radar
+10:00-10:15 | **Zhenhua Hao** - Different migratory strategies in locusts and moths revealed by radar **remote**{: .badge .bg-secondary }
+10:15-10:30 | **Xu Shi** - Prospects for monitoring bird migration along the East Asian-Australasian Flyway using weather radar **speed talk**{: .badge .bg-secondary } **remote**{: .badge .bg-secondary }
+&nbsp;      | **Lili Jia** - Using radar to mitigate birdstrike risk in coastal airport on Bohai Bay **speed talk**{: .badge .bg-secondary } **remote**{: .badge .bg-secondary }		
+10:30-11:00 | Coffee break
+**Session 4**{: .text-primary } | **From radar aeroecology to biodiversity**{: .text-primary }
+11:00-11:15 | **Andrew Farnsworth** - From data to action: BirdCast perspectives on transforming bird migration science to conservation planning
+11:15-11:30	| **Hans van Gasteren** - Validation of local bird radars used in military aviation
+11:30-11:45	| **Arndt Wellbrock** - Flying across the sea – possibilities and limitations of weather radar data for the understanding of bird migration over the Baltic Sea **remote**{: .badge .bg-secondary }
+11:45-12:00	| **Maja Bradarić** - On the radar: offshore wind turbine curtailment informed by nocturnal bird migration predictions
+12:00-13:30 | Lunch break
+**Session 3**{: .text-primary } | **Applied radar aeroecology - the knowledge base for conservation and management**{: .text-primary }
+13:30-14:00 | **Gao Bayo** - Migration basics insects **keynote**{: .badge .bg-secondary } **remote**{: .badge .bg-secondary }
+14:00-14:15	| **Theodore Zenzal** - A Bioenergetic Model to Facilitate Migratory Landbird Conservation along the Northern Gulf of Mexico Coast **remote**{: .badge .bg-secondary }
+14:15-14:30	| **Elske Tielens** - Spatial patterns in diurnal aerial insect biomass and landscape level drivers across the continental United States
+14:30-14:45	| **Adriaan Dokter** - Macro-demography of North America’s migratory birds
+14:45-15:15 | Coffee break
+15:15-15:30	| **Raphaël Nussbaumer** - Aerial and terrestrial biomass flows of migratory birds across the US
+15:30:15:45	| **Christopher Hassall** - Weather surveillance radar for biodiversity monitoring in the UK
+15:45-16:00	| **Jason Chapman** - Long-term trends of insect migration patterns above the USA
+16:00-16:30	| **Silke Bauer & Judy Shamoun-Baranes** - Concluding talk 
+17:00+      | Drinks & poster session
```

Comment:
Don't see the poster here in the code, but Brian Lee's poster should be taken out. He retracted his contribution after sending the invite.

Reply:
Done!